src/fortune-wheel-booth-frontend/src/costants.ts

```diff
@@ -0,0 +1,60 @@
+import Duck from './assets/images/duck.png';
+import Lemon from './assets/images/lemon.png';
+import ckBtc from './assets/images/ckbtc.png';
+import ckEth from './assets/images/cketh.png';
+import IcpLogoLight from './assets/images/icp-logo-light.png';
+import { WheelDataType } from 'react-custom-roulette';
+
+export const PRIZES: WheelDataType[] = [
+  {
+    option: 'ckEth0_01',
+    image: { uri: ckEth, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#FBB03C' },
+  },
+  {
+    option: 'icp1',
+    image: { uri: IcpLogoLight, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#7B2582' },
+  },
+  {
+    option: 'ckEth0_05',
+    image: { uri: ckEth, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#28ABE2' },
+  },
+  {
+    option: 'icp0_5',
+    image: { uri: IcpLogoLight, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#F36526' },
+  },
+  {
+    option: 'ckBtc0_001',
+    image: { uri: ckBtc, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#564B9E' },
+  },
+  {
+    option: 'merchHat',
+    image: { uri: Duck, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#1EED3E' },
+  },
+  {
+    option: 'ckBtc0_005',
+    image: { uri: ckBtc, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#ED1E78' },
+  },
+  {
+    option: 'merchTshirt',
+    image: { uri: Lemon, sizeMultiplier: 0.8, offsetY: 140 },
+    style: { backgroundColor: '#FFEC0C' },
+  },
+];
+
+export const PRIZES_VALUES_MAPPING: Record<string, string> = {
+  ckEth0_01: '0.01 ckEth',
+  icp1: '1 icp',
+  ckEth0_05: '0.05 ckEth',
+  icp0_5: '0.5 icp',
+  ckBtc0_001: '0.001 ckBtc',
+  merchHat: 'Hat',
+  ckBtc0_005: '0.005 ckBtc',
+  merchTshirt: 'T-Shirt',
+};
```

Comment:
Why can't these values be inside the prize directly? E.g. using a `displayName` key

Reply:
We don't want to touch the `WheelDataType`, so fine to keep the display names here. We should capitalize the token names though (e.g. Btc -> BTC)